ui/de-lib/src/main/java/org/iplantc/de/apps/integration/client/view/propertyEditors/widgets/ArgumentValidatorEditor.java

```diff
@@ -338,4 +338,10 @@ void onEditButtonSelected(@SuppressWarnings("unused") SelectEvent event) {
         dlg.show();
     }
 
+    List<ArgumentValidator> getValidators() {
+        List<ArgumentValidator> values = Lists.newArrayList();
+        values.addAll(validatorStore.getAll());
+        return values;
+    }
```

Comment:
Was it necessary to copy the unmodifiable list of validators to the event because of this call above?
```java
model.setValidators(event.getValue());
```